scripts/build_wheel.py

```diff
@@ -52,6 +52,12 @@
       long_description=long_description,
       long_description_content_type="text/markdown",
       url="https://github.com/python/typeshed",
+      project_urls={{
+          "GitHub": "https://github.com/python/typeshed",
```

Comment:
While this is a duplicate of the generic url field, it adds a GitHub icon on PyPI, which can be helpful.